packages/flutter/lib/src/material/tab_indicator.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+
+/// Used with [TabBar.indicatorShape] to draw a horizontal line below the
+/// selected tab
+///
+/// The line is inset from the tab's boundary by [padding]. The [borderSide]
+/// defines the line's color and weight.
+///
+/// The [TabBar.indicatorSize] property can be used to define the
+/// tab's boundary in terms of its (centered) widget, [TabIndicatorSize.label],
+/// or the the entire tab, [TabIndicatorSize.tab].
+class DefaultTabIndicator extends ShapeBorder {
```

Comment:
i would call this an UnderlineTabIndicator, so that if we ever want to change the default we're not stuck with a weird class name

Reply:
Good idea, done.